Sources/NostrSDK/Events/Calendars/CalendarEventRSVPFreebusy.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CalendarEventRSVPFreebusy.swift
+//
+//
+//  Created by Terry Yiu on 12/17/23.
+//
+
+import Foundation
+
+/// Determines if the user would be free or busy for the duration of the calendar event.
+public enum CalendarEventRSVPFreebusy: RawRepresentable, CaseIterable, Codable, Equatable {
+
+    public typealias RawValue = String
+
+    /// The user is free for the duration of the calendar event.
+    case free
+
+    /// The user is busy for the duration of the calendar event.
+    case busy
+
+    /// Unknown freebusy state.
+    case unknown(RawValue)
```

Comment:
"freebusy" seems like there are only two possibilities. Is it even possible that this could have more cases in the future? If not, the enum simplifies significantly.

Reply:
The current NIP supports only free or busy. But the iCal spec it was based on also supports busy-unavailable and busy-tentative. I think anything is possible, but I'm leaving this unknown here because anyone could shove a random value into this fields that's unrecognized with a different client.

https://datatracker.ietf.org/doc/html/rfc5545#section-3.2.9